src/Datadog.Trace.ServiceFabric/Remoting.cs

```diff
@@ -389,5 +366,19 @@ private static void FinishSpan(EventArgs e, string spanKind)
             bool analyticsEnabled = integrationSettings.AnalyticsEnabled ?? (enabledWithGlobalSetting && tracer.Settings.AnalyticsEnabled);
             return analyticsEnabled ? integrationSettings.AnalyticsSampleRate : (double?)null;
         }
+
+        private static IServiceFabricContextPropagator GetContextPropagator(Tracer instance)
```

Comment:
I think this should actually reuse existing context agnostic propagator. 
See experimental code [here](https://github.com/RassK/opentelemetry-dotnet-instrumentation/compare/context-propagation-b3...RassK:context-propagation-b3-experimental)